tests/terraform/createcluster/createcluster_test.go

```diff
@@ -26,84 +23,101 @@ func Test_TFClusterCreateValidation(t *testing.T) {
 }
 
 var _ = Describe("Test:", func() {
+
 	Context("Build Cluster:", func() {
 		It("Starts up with no issues", func() {
-			status, err := terraform.BuildCluster(&testing.T{}, *tfVars, false)
+			status, err := BuildCluster(&testing.T{}, false)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(status).To(Equal("cluster created"))
+
 			defer GinkgoRecover()
-			fmt.Println("Server Node IPS:", terraform.MasterIPs)
-			fmt.Println("Agent Node IPS:", terraform.WorkerIPs)
-			terraform.PrintFileContents(terraform.KubeConfigFile)
-			Expect(terraform.MasterIPs).ShouldNot(BeEmpty())
-			if terraform.NumWorkers > 0 {
-				Expect(terraform.WorkerIPs).ShouldNot(BeEmpty())
+
+			fmt.Println("Server Node IPS:", MasterIPs)
+			fmt.Println("Agent Node IPS:", WorkerIPs)
+			fmt.Println(KubeConfigFile)
```

Comment:
We should keep the `PrintFileContents` from this, otherwise it will just print the path to the kubeconfig. Printing the contents is what we want so that we can copy the kubeconfig to our local `./kube/config` and perform any kubectl debugging commands locally without ssh'ing into the VMs.